### ISSUE TITLE
JAWS does not announce the application role to users

### ISSUE THREAD

**Issue by dabrams888** [FreedomScientific/VFO-standards-support#609]
### Summary

> When navigating into an element with role="application" JAWS does not expose the role information to the user. If the application has an accessible name via the aria-label attribute the name is announced, but no corresponding role information is announced with it.

### Expected result

> JAWS switches to forms mode and announces the application's role and name information.

### Actual result

> JAWS switches to forms mode and announces the application's name, but not its role.

### Example

> 1. Go to https://codepen.io/dabrams888/full/ZEagdMG
> 2. Tab to the "Focus me" button that is contained within an application

#### JAWS version and build number
Version 2022.2112.24 ILM

 
#### Operating System and version
Microsoft Windows 10 Pro x64
Version 10.0.19044 Build 19044

#### Browser and version:
Chrome Version 99.0.4844.51 (64-bit)

**Comment by JAWS-test** [comment]
The change between reading mode, forms mode, and application mode is indicated by a beep in JAWS. This works correctly in your example.

When reading with the virtual cursor, the beginning and end of the application area is explicitly output in text form. However, only if this has not been deactivated in the settings. This also works correctly in your example.

Therefore, I would say that there is no bug and the issue can be closed.

**Comment by dabrams888** [comment]
@JAWS-test can you clarify where in settings the text output would be specified? I am using the default settings and do not hear the role announced when entering/exiting the application area. I only hear the beep.

**Comment by JAWS-test** [comment]
INS+6 (JAWS Setting Center): Web > Reading > Web Verbosity Level > Configure Web Verbosity Level > Regions: Application
